sdk/formrecognizer/azure-ai-formrecognizer/src/test/java/com/azure/ai/formrecognizer/documentanalysis/DocumentAnalysisClientTest.java

```diff
@@ -1676,7 +1678,8 @@ public void testClassifyAnalyze(HttpClient httpClient,
                     .setPollInterval(durationTestMode);
                 AnalyzeResult analyzeResult = syncPoller.getFinalResult();
                 Assertions.assertNotNull(analyzeResult);
-                Assertions.assertEquals(2, analyzeResult.getDocuments().size());
+                Assertions.assertEquals(3, analyzeResult.getDocuments().size());
+                Assertions.assertEquals(analyzeResult.getModelId(), classifierId);
             }, IRS_1040);
         }
     }
```

Comment:
Did you already have the formula, barcode, etc tests in this file?

Reply:
no they were added as a seperate test